### DISCUSSION TITLE
Specified request cannot be processed.

### DISCUSSION THREAD

**Discussion by dotsinspace** [troublediehard/magento-react-native#170] [General]
Hello i am new to magento but have well depth knowledge about react native...i have gone through process about integration
of app in magento dashboard to swagger about apis available. now i am in trouble since morning due to this error. Specified Request cannot be processed after putting website address in magento.js under config folder. as follows.

1. Throws error ( Specified request cannot be processed ).
`
export const magentoOptions = {
  url: 'http://aladainexpress.com/', 
  home_cms_block_id: '235',
  store: 'main_website_store',
  authentication: {
    integration: {
      access_token: 'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'
    },
  },
  reviewEnabled: false, // set true ONLY if you install https://github.com/troublediehard/mma-customapi on the magento backend
};
`

2. This works but throws JSON parse error Unexpected token: <
`
export const magentoOptions = {
  url: 'http://aladainexpress.com/main_website_store/', // make sure you have trail slash in the end
  home_cms_block_id: '235',
  store: 'main_website_store', // store code // Stores > All Stores > Store View > Code
  authentication: {
    integration: {
      access_token: 'bqp7h58qfa3q7cyctx8oxnmp6thw0bit'
    },
  },
  reviewEnabled: false, // set true ONLY if you install https://github.com/troublediehard/mma-customapi on the magento backend
};
`

**Comment by dimaportenko** [comment]
I've just tried following config, and it's working on my end
```
export const magentoOptions = {
  url: 'http://aladainexpress.com/',
  home_cms_block_id: '235',
  store: 'default',
  authentication: {
    integration: { access_token: 'bqp7h58qfa3q7cyctx8oxnmp6thw0bit' },
  },
  reviewEnabled: false,
};
```

**Reply by dimaportenko** [reply]
@dotsinspace where did you get value `main_website_store`?

**Reply by dotsinspace** [reply]
from magento2 dashboard in store section

**Reply by dimaportenko** [reply]
can you please share the screenshot?
